### ISSUE TITLE
Click on the coordinates

### ISSUE THREAD

**Issue by ziceptor** [Arckenver/MightyLoot#8]
When you click on the coordinates...
[pic](https://forums-cdn.spongepowered.org/uploads/default/original/3X/d/d/dd02d2aecc35d25a7579f455d98b2cc7b957f78d.png)

I would like to have the configuration in the config. For example: command=... {POS}. To change the output format {POS} x;y;z -> x y z

The owner of the server would have figured out how to use it. /tp or /broadcast and etc. Or add a separate command for this.

**Comment by Arckenver** [comment]
Good idea. For now I'll consider it low-priority feature but I'll keep the issue open until it's done.
